tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py

```diff
@@ -17,6 +17,8 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         self.connect_to_local_area_network()
         self.apps.set_permission_by_url(Search.manifest_url, 'geolocation', 'deny')
+        # not sure why the Smart Collection geolocation is being prompted.
+        self.apps.set_permission_by_url(Collection.manifest_url, 'geolocation', 'deny')
```

Comment:
As far as I know they removed the geolocation prompt from the search app and possibly from the collections too. I was not able to get a prompt on my device manually.

Also, when sunning the test it failed at this step with the following error:
```
TEST-UNEXPECTED-ERROR | test_browser_bookmark.py TestBrowserBookmark.test_browser_bookmark | JavascriptException: JavascriptException: NS_ERROR_UNEXPECTED: 
stacktrace:
	execute_async_script @gaia_test.py, line 55
	inline javascript, line 99
	src: "      mozPerms.set("



Traceback (most recent call last):
  File "/home/robertchira/.virtualenvs/gaia/local/lib/python2.7/site-packages/marionette_client-0.8.5-py2.7.egg/marionette/marionette_test.py", line 249, in run
    self.setUp()
  File "/home/robertchira/workspace/gaia/tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py", line 19, in setUp
    self.apps.set_permission_by_url(Search.manifest_url, 'geolocation', 'deny')
  File "/home/robertchira/workspace/gaia/tests/python/gaia-ui-tests/gaiatest/gaia_test.py", line 55, in set_permission_by_url
    (manifest_url, permission_name, value))
  File "/home/robertchira/.virtualenvs/gaia/local/lib/python2.7/site-packages/marionette_client-0.8.5-py2.7.egg/marionette/marionette.py", line 1285, in execute_async_script
    filename=os.path.basename(frame[0]))
  File "/home/robertchira/.virtualenvs/gaia/local/lib/python2.7/site-packages/marionette_client-0.8.5-py2.7.egg/marionette/decorators.py", line 36, in _
    return func(*args, **kwargs)
  File "/home/robertchira/.virtualenvs/gaia/local/lib/python2.7/site-packages/marionette_client-0.8.5-py2.7.egg/marionette/marionette.py", line 630, in _send_message
    self._handle_error(response)
  File "/home/robertchira/.virtualenvs/gaia/local/lib/python2.7/site-packages/marionette_client-0.8.5-py2.7.egg/marionette/marionette.py", line 678, in _handle_error
    raise errors.JavascriptException(message=message, status=status, stacktrace=stacktrace)
TEST-INFO took 67995ms

```